templates/vstore_dashboard_template.php

```diff
@@ -160,11 +160,12 @@
  * List downloads
  */
 $VSTORE_DASHBOARD_TEMPLATE['address']['view'] = '
-	<div class="row">
+
+<div class="row">
 	    <div class="col-12 col-xs-12 col-sm-6">
 	    	<div class="card h-100">
 		        <div class="card-body">
-			        <h4 class="card_title">Billing</h4>
+			        <h4 class="card_title">'.LAN_VSTORE_GEN_017.'</h4>
```

Comment:
For all template files use: `{LAN=VSTORE_XXX}` (without the `'. .' `) which will render `LAN_VSTORE_XXX`

eg. instead of this: 
`<h4 class="card_title">'.LAN_VSTORE_000.'</h4>`
use this: 
`<h4 class="card_title">{LAN=VSTORE_000}</h4>`

Sorry for all the changes, but it's important we get it right for a clean release.